src/content/components/announcement-banner.md

```diff
@@ -3,4 +3,4 @@ template: announcement
 title: Announcement
 ---
 
-Maximize your BTC in DeFi. <a href="https://dashboard.threshold.network/tBTC/mint" target="_blank" rel="noopener noreferrer">Mint tBTC now</a>
+Maximize your BTC in DeFi. <a href="https://dashboard.threshold.network/tBTC/mint" target="_blank" rel="noopener noreferrer">Mint tBTC now</a> and use tBTC as collateral for <a href="https://thresholdusd.org/" target="_blank" rel="noopener noreferrer">thUSD stablecoin</a>
```

Comment:
This change causes the navigation bar to extend beyond the right edge of the screen.

<img width="1506" alt="image" src="https://github.com/threshold-network/website/assets/40306834/368b8cc3-d0ad-45f0-bd42-ef6f1f16a827">

1154px width:
<img width="1156" alt="image" src="https://github.com/threshold-network/website/assets/40306834/ab722ed5-453a-49a0-8ad9-302205317dd8">

To apply such chage we should probably rewrite the navigation panel in a more scalable way.

Reply:
I took a look -- unfortunately frontend work is not my strong suit and I'm at a loss on how to fix this.  @michalsmiarowski @evandrosaturnino let me know if you have suggestions, thanks.

Reply:
Personally I don't think a thusd button should be added horizontally here.

if we are going to update this section, I think an entire refactor should be done, adding a products menu to the navbar, that would expand a panel with the existing products links, icons and descriptions: thUSD, tBTC and TAco. 

But this requires a design work. So for now I would just revert this button.

Reply:
Thanks @evandrosaturnino I agree.  Will push an update.

Reply:
@evandrosaturnino @michalsmiarowski reverted nav-bar, please have another look so we can get this to main.  Thank you.

Reply:
Just for future reference: I've marked a wrong lines of code here. I was talking about [this](https://github.com/threshold-network/website/pull/118/commits/db749be6de5512b5cd3109965040eff707c11e9d#diff-9c9bdbcbf649e13217df99aa87eee7caeae5ea8b3aac4f14530fdb6aa4a2965dR47-R48), which ineed was reverted 😉